modules/ROOT/pages/index.adoc

```diff
@@ -3,12 +3,76 @@
 :idseparator: -
 :!example-caption:
 :!table-caption:
-
-You can get started right away by downloading the {sdk-long-name} and cloning an existing sample project or by exploring the documentation. Check out the tiles below to begin building open services and software to run on the {IC}.
+:page-layout: landing
 
 ++++
 
 <div class="html-container">
+
+    <div class="landing-hero">
+        <h1>The next generation of software engineering starts with you.</h1>
+        <div class="landing-cta">
+        <p>To download the DFINITY Canister SDK, run the following command in your terminal.</p>
+++++
+
+[source,bash]
+----
+sh -ci "$(curl -fsSL https://sdk.dfinity.org/install.sh)"
+----
+
+++++
+        <p>By installing the SDK, you agree to our <a href="https://sdk.dfinity.org/sdk-license-agreement.txt">Terms & Conditions</a>.</p>
+        </div>
+    </div>
+
+<div class="home-pitches-grid">
+
+    <div class="landing-pitch">
+        <img src="_images/HelloWorld.png" height="320" width="320" />
+        <h2>What can you imagine?</h2>
+        <p>Decentralized software can be more than financial transactions. The Internet Computer is a general purpose platform and our developer community has built everything from video chat to mobile games to compilers.</p>
+        <p class="link"><a href="http://github.com/dfinity/awesome-dfinity">Check out some awesome apps</a></p>
+    </div>
+
+    <div class="landing-pitch">
+        <img src="_images/CorporateCloud.png" height="320" width="320" />
+        <h2>Escape the Corporate Cloud</h2>
+        <p>Why build decentralized software just to run your front-end on monopolistic hardware like Amazon Web Services or Google Cloud? Build your front-end in any framework (React, Angular, Vue, or even vanilla JavaScript) and your users can access it from a standard browser — <b>no plug-ins</b>.</p>
+        <p class="link"><a href="https://sdk.dfinity.org/docs/developers-guide/webpack-config.html">Read more about front-end development</a></p>
+    </div>
+
+    <div class="landing-pitch">
+        <img src="_images/MotokoLogo.png" height="320" width="320" />
+        <h2>Your new favorite language</h2>
+        <p>The <b>Motoko</b> language is optimized for authoring for the Internet Computer and packed with so many familiar features you may forget you're working with a blockchain: async/await, randomness, time and date, and even floating point numbers. </p>
```

Comment:
```suggestion
        <p>The <b>Motoko</b> language is optimized for authoring for the Internet Computer yet packed with familiar features—like async/await, randomness, time and date, and even floating point numbers—you might forget you're working with a blockchain.</p>
```

Reply:
I'm not sure I like my suggestion, but it felt awkward to me to have the list of features after the blockchain colon.